src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.spec.ts

```diff
@@ -22,15 +29,66 @@ describe('RgwMultisiteSyncPipeModalComponent', () => {
         ReactiveFormsModule,
         CommonModule
       ],
-      providers: [NgbActiveModal]
+      providers: [NgbActiveModal, { provide: RgwMultisiteService, useValue: multisiteServiceMock }]
     }).compileComponents();
 
     fixture = TestBed.createComponent(RgwMultisiteSyncPipeModalComponent);
     component = fixture.componentInstance;
+    multisiteService = TestBed.inject(RgwMultisiteService);
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should replace * with All Zones (*)', () => {
```

Comment:
We can use `` ` ` ``  or `" "` here :
```suggestion
  it("should replace '*' with 'All Zones (*)' ", () => {
```
Same for below ones.